db/migrate/20250618000304_add_call_for_papers_to_events.rb

```diff
@@ -0,0 +1,6 @@
+class AddCallForPapersToEvents < ActiveRecord::Migration[8.0]
+  def change
+    add_column :events, :call_for_papers_link, :string
+    add_column :events, :call_for_papers_deadline, :date
```

Comment:
It would be great to also have a date for when the CFP opens. Sometimes a CFP gets announced but isn't immediately open.
```suggestion
    add_column :events, :call_for_papers_link, :string
    add_column :events, :call_for_papers_open_date, :date    
    add_column :events, :call_for_papers_deadline, :date
```

Also, while we are here: on RubyConferences we used:
* `cfp_link`
* `cfp_open_date`
* `cfp_close_date`

I wonder if that is more compact and familiar when we'd use the same column names. I also like that the date columns have a `*_date` postfix, so you know what type it is.